src/System.Security.Permissions/src/System/Diagnostics/EventLogPermissionEntryCollection.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+namespace System.Diagnostics
+{
+    [Serializable]
+    public class EventLogPermissionEntryCollection : CollectionBase
+    {
+        internal EventLogPermissionEntryCollection() { }
```

Comment:
The internal constructor combined with the Serialize test creating a uninitialized object type causes the Innerlist to be null. It seems like the test should skip these kinds of classes so I'm heading down that route at the moment.